modules/extra/joke.rb

```diff
@@ -26,5 +26,21 @@ module Extra
           event.respond("Confucious say #{File.readlines('text/Jokes/confucious.txt').sample.chomp}")
         }
     )
+
+    $cbot.add_command(:notice,
+        code: proc { |event, args|
+          if args.length < 1 || args[0] == 'me'
+            whom = event.user.name
```

Comment:
For future code maturity, consider using `whom'st've`.

Reply:
no